src/index.html

```diff
@@ -15,8 +15,68 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="http://">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="ul">
+          <li class="li">
+            <a
+              href="http://"
+              class="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="li">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="li">
```